wcc-contentful-app/app/views/sections/_video_highlight.html.erb

```diff
@@ -10,8 +10,8 @@
       </div>
     <% end %>
   </div>
-  <div class="column medium-6">
-    <div class="video-responsive section-video-highlight__video">
+  <div class="section-video-highlight__video col-xs-12 col-md-6">
```

Comment:
question - were we putting the `__` convention into the gem, or could we use the gem as an opportunity to use nesting convention (rewrite some of the WMR convention)?

Reply:
I don't think we've discussed whether we will change conventions.  Would you like to put something on the calendar?